src/shared-components/verificationMessages/formatMessage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import uniqueid from 'lodash.uniqueid';
+
+import { messageType } from '..';
+
+const formatMessage = (message: messageType) => {
+  if (Array.isArray(message)) {
+    const mapResult = message.map((value) => (
+      <React.Fragment key={uniqueid()}>{value}</React.Fragment>
+    ));
+
+    return (
+      <React.Fragment>
+        {mapResult.reduce((prev, curr) => (
+          <React.Fragment>
+            {prev}
```

Comment:
the old syntax had TS errors, I tested this in the Story and works also added a test.